extensions/classification.md

```diff
@@ -0,0 +1,48 @@
+# Information Classification
+
+As part of ISO27001 control objective A8.2 aims to address 'Information
+Classification' whereby information and data in an organization is properly
+managed, including classification in relation to sensitivity of the data,
+legislation etc. A.8.2.2 requires that electronic assets should be 'labeled',
+and this extension allows the `data` of cloudevents to be appropriately labeled
+with the `classification` of the event being shared.
+
+organizations will typically have their own internal Information Management
+policies and standards that might address these control objectives differently,
+however this extension should provide a consistent framework for labeling
+events, where organization can use their own internal values for
+`classification`.
+
+This extension is not opinionated on how this classification will be used and
+does for example imply different encryption schemes etc.
+
+For Sainsburys-tech colleagues, additional information can be found on our
+internal confluence under `/display/IC/Information+Classification+Extension`,
+which for example includes the values from our own classification policies.
+
+## Attributes
+
+### classification
+
+- Type: `String`
+- Description: The `classification` of the `data`. The value SHOULD be expressed
+  in uppercase.
```

Comment:
Why put a constraint on the uppercase part, I think the classification labels should be open?

Reply:
I wasn't sure here, I figured it was better to assume case-insensitivity as "public" and "PUBLIC" for example should be treated equally to avoid potential confusions.  It's also common practice in a few organizations I've been in to use uppercase on documents as it makes it clear the word is used as a 'keyword, in relation to policy'. Perhaps we could define the values should be treated case-insensitive, but allow the value set either way? I was leaning to force upper as it makes it explicit, and the 'SHOULD' allows flexibility if for some reason it were problematic.